.pre-commit-config.yaml

```diff
@@ -1,44 +1,59 @@
 repos:
 -   repo: https://github.com/python/black
-    rev: 19.10b0
+    rev: 20.8b1
```

Comment:
Won't this version of black report errors on 1.1.x?

Reply:
You're probably right. Actually I was noticing some weirdness with black as well, so perhaps better just to `--no-verify`